blockchain/fundamentals/anonimity/how-to-de-anonymize-bitcoin.md

```diff
@@ -0,0 +1,59 @@
+# How to De-Anonymize Bitcoin
+author: mihaiberq
+
+levels:
+
+  - basic
+
+  - medium
+
+type: normal
+
+category: must-know
+
+tags:
+
+  - introduction
+
+  - deep
+
+links:
+  - '[Bitcoin transactions are not that anonymous.](https://www.technologyreview.com/s/608716/bitcoin-transactions-arent-as-anonymous-as-everyone-hoped/)'
+
+---
+## Content
+
+In the previous insight, we mentioned a specific way of linking transactions to an online identity: joint payments. Most of the times, however, a single redeemed UTXO is enough to track someone if the transacted value is specific enough.
+
+A bitcoin transaction can represent up to 8 digits after the decimal points (`0.000000001 bitcoin` is called a `satoshi`), so the distribution space is large. If someone is to know the exact value of your transaction and a timeframe in which you had intiated the transaction, there's a chance they could track your address.
```

Comment:
Note here what the odds of someone exchanging the name exact decimal value in an hour, given the maximum number of transactions the bitcoin network can process per hour, and how many hours of time need to elapse before that value gets close enough to 1 to de-anonymize you